### ISSUE TITLE
iOS: Biometrics don't work on iPhone 13 Pro

### ISSUE THREAD

**Issue by mpfau** [tutao/tutanota#3759]
**Bug in mobile app**
User on iPhone 13 Pro reports that the option to enable the new Biometric/PIN login is not displayed in their Login Menu.

**To Reproduce**

* Go to Settings > Login
* Credentials Encryption Mode is not listed

**Expected behavior**

**Smartphone (please complete the following information):**
 - Device: iPhone 13 Pro
 - OS: iOS 15.2
 - Version  3.89.26

**Additional context**
Everything works as expected on an iPhone 12 Pro with Tutanota 3.89.26 and iOS 15.2

**Comment by Jack15911** [comment]
Does not work in my iPhone 8 running iOS 15.3.1: "credential encryption mode" does not display in iOS Tutanota Settings->Login. Downloaded from Apple's App Store. Deleted app from phone and re-installed. Still does not function.

**Comment by charlag** [comment]
@Jack15911 do you have biometric added in iOS settings?

**Comment by Jack15911** [comment]
I'm using biometric 2FA on other apps, so yes to those. If Tutanota requires it specifically in the Tutanota app, then I haven't found it, so no.

**Comment by Jack15911** [comment]
> @Jack15911 do you have biometric added in iOS settings?

I'm using biometric 2FA in other apps, so yes. If the Tutanota app requires specific approval in their app, I haven't been able to locate it, so in that case, no.

**Comment by charlag** [comment]
@Jack15911 no, it doesn't. That's quite odd but we had other reports about that.
Question: did you install it with iOS 15 already or did you upgrade? I have a feeling that after upgrade to one or another iOS versions biometrics don't work anymore

**Comment by charlag** [comment]
This doesn't seen to be specific to any particular iPhone model but rather happens across any iOS 15 device. As far as I know reinstall helps. I bet Apple broke something during upgrade.
